test/export.jl

```diff
@@ -31,7 +31,9 @@
     @testset "Record metadata" begin
         function change_sample_rate(samples; sample_rate)
             info = SamplesInfoV2(Tables.rowmerge(samples.info; sample_rate=sample_rate))
-            new_data = similar(samples.data, 0, Onda.index_from_time(sample_rate, Onda.duration(samples)) - 1)
+            # sparse CSC but for wide data we have a huge column pointer
+            # so transposing to get sparse CSR
+            new_data = spzeros(eltype(samples.data), Onda.index_from_time(sample_rate, Onda.duration(samples)) - 1, channel_count(samples))'
```

Comment:
I don't understand this change; AFAICT it's not really related to the EDF.jl changes, but maybe I'm missing something?

Reply:
@palday did you mean to push this change here?

Reply:
tests [were failing on 1.6](https://github.com/beacon-biosignals/OndaEDF.jl/actions/runs/13020881507/job/36321044154#step:7:250) because whatever Onda version gets resolved there ignored `validate=false` when using the old 0d array. Sparse matrices avoid that and the allocation of a massive dense array.

Reply:
we can drop 1.6 support IMO, it's not longer LTS

Reply:
I'm also fine with that, whether or not you leave the sparse change in.

Reply:
Turns out this is still a problem in 1.10, so I'm going to keep @palday's change.

Reply:
I'm so sorry. I really didn't mean to bring this upon you!

Reply:
😆